moduleframework/timeoutlib.py

```diff
@@ -60,9 +51,8 @@ def __enter__(self):
     def __exit__(self, *args, **kwargs):
         pass
 
-
 class Retry(object):
-    def __init__(self, attempts=1, timeout=None, exceptions=(Exception,), error=None, inverse=False, delay=None):
+    def __init__(self, attempts = 1, timeout = None, exceptions = (Exception,), error = None, inverse = False, delay = None):
```

Comment:
this doesn't seem to be according PEP8 recommendations

Reply:
I'll repair it. I've done hardcopy of https://github.com/cockpit-project/cockpit/blob/master/test/avocado/timeoutlib.py, to remove own differences